client/src/i18n/fr/stock.json

```diff
@@ -74,9 +75,9 @@
     "FROM"            : "Provenance",
     "GROUPING"        : "Groupage",
     "JOIN_INVOICE"    : "Joindre une facture",
-    "INCLUDE_EMPTY_LOTS" : "Inclure les lots épuisés",
+    "INCLUDE_EMPTY_LOTS" : "Inclure les articles non en stock",
```

Comment:
I believe this is is referring to https://github.com/IMA-WorldHealth/bhima/issues/5322 which it would address.  However, I'm not sure that it isn't used elsewhere to actually mean lots.

If you are fixing https://github.com/IMA-WorldHealth/bhima/issues/5322, could you create a new key for articles that are out of stock and use that?

Reply:
Done.  Although there is an issue here that should be fixed in another PR.  When I made the bhStockFilterExhausted component (see the Default filters in Stock Lots), There is a (probably left-over) reference to INCLUDE_EMPTY_LOTS_HELP which does not seem to be working.   See the end bhStockFilterExhausted.tmp.html.  If you concur that it is no longer needed (which is what I thin), I'll delete it and rebase this PR before releasing.

Reply:
> Although there is an issue here that should be fixed in another PR. When I made the bhStockFilterExhausted component (see the Default filters in Stock Lots), There is a (probably left-over) reference to INCLUDE_EMPTY_LOTS_HELP which does not seem to be working. See the end bhStockFilterExhausted.tmp.html. If you concur that it is no longer needed (which is what I thin), I'll delete it and rebase this PR before releasing.

I'll deal with this is a separate PR.